app.py

```diff
@@ -0,0 +1,127 @@
+from dash import Dash, html, Input, Output, dcc, ctx, State
+import dash_bootstrap_components as dbc
+import dash_cytoscape as cyto
+
+# Import the module data as a dataframe
+import module_data
+df = module_data.df
+
+# Import app components and their internal callbacks
+from components.left_hand_nav_bar import left_hand_nav_bar, left_hand_nav_bar_callbacks 
+left_hand_nav_bar = left_hand_nav_bar.left_hand_nav_bar
+
+from components.visualization_panels import combined_visualization_panel, pathway_visualization, active_module_visualization, search_results_visualization
+combined_visualization_panel = combined_visualization_panel.combined_visualization_panel
+pathway_visualization = pathway_visualization.pathway_visualization
+active_module_visualization = active_module_visualization.active_module_visualization
+search_results_visualization = search_results_visualization.search_results_visualization
+
+from components.app_title import app_title
+app_title = app_title.app_title
+
+from components.clickable_module_list import clickable_module_list, clickable_module_list_callbacks
+clickable_module_list_panel = clickable_module_list.clickable_module_list
+
+from components.module_details_panel import module_details_panel, module_details_panel_callbacks
+module_information = module_details_panel.module_details_panel
+
+from components.left_hand_nav_bar import search_panel
+search_panel = search_panel.search_panel
```

Comment:
The imports where it's assigning to the same name I *think* are unnecessary. I'm pretty sure `from components.left_hand_nav_bar import search_panel` will let you reference `search_panel` without this line.